### PR TITLE
Integrations content - Master PR

### LINKED ISSUE #2378: [Feedback] Page: /use-timescale/latest/ingest-data/ingest-prometheus/
### Is it easy to find the information you need?

No

### Are the instructions clear?

No

### How could we improve the Timescale documentation site?

uh this has no details on how to get data scraped from a prometheus compatible endpoint into timescaledb.  it just says what prometheus is

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Sets up the new integrations content page structure. Individual sections can be moved and edited in PRs against this one.

# Links

- Fixes: https://github.com/timescale/docs-private/issues/99
- Fixes: https://github.com/timescale/docs-private/issues/117
- Fixes: https://github.com/timescale/docs/issues/2378

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: use-timescale/integrations/observability-alerting/grafana/create-dashboard-and-panel.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+1.  You can edit the query directly, or use the built-in query editor.
+
+    <Highlight type="note">
+    If you are visualizing time series data in Grafana, make sure you select
+    `Time series` from the `Format As` drop down in the query builder.
+    </Highlight>
+
+</Procedure>
+
+## The Grafana timeFilter function
```
Comment (Loquacity) [inline use-timescale/integrations/observability-alerting/grafana/create-dashboard-and-panel.md:46]:
```suggestion
## The time filter function
```

**Comment by gina-alioto** [comment]
Hi @Loquacity The link for Grafana isn't working.

**Comment by Rajakavitha1** [comment]
Hi @charislam !!! I checked all the links and the content they seem fine to me. Except that I get a 403 for https://docs-dev.timescale.com/docs-dev-integrations-setup/use-timescale/dev-integrations-setup/integrations/query-admin/troubleshooting/. My understanding is that the Troubleshooting section cloud-jdbc-authentication-support should be automatically pulled in under the TS section.

**Review by Rajakavitha1 — APPROVED** [review]
LGTM!!!
only the Troubleshooting section link throws a 403.

**Comment by gina-alioto** [comment]
Hi @Loquacity I had a look as well and I think it's just fine for now.  Just fix the 403 (Troubleshooting) and we're initially good!

**Review by gina-alioto — APPROVED** [review]

**Pull request merged**